Loop/Managers/Missed Meal Detection/MissedMealSettings.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  MissedMealSettings.swift
+//  Loop
+//
+//  Created by Anna Quinlan on 11/28/22.
+//  Copyright © 2022 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+
+public struct MissedMealSettings {
+    /// Minimum grams of unannounced carbs that must be detected for a notification to be delivered
+    public static let minCarbThreshold: Double = 15 // grams
+    /// Maximum grams of unannounced carbs that the algorithm will search for
+    public static let maxCarbThreshold: Double = 80 // grams
+    /// Minimum threshold for glucose rise over the detection window
+    static let glucoseRiseThreshold = 2.0 // mg/dL/m
+    /// Minimum time from now that must have passed for the meal to be detected
+    public static let minRecency = TimeInterval(minutes: 25)
```

Comment:
Other places in Loop, like momentum calculation, we look at an average or regression over multiple samples to reduce noise, and use a shorter period, like 15 minutes.  And from my own personal experience, if I see 3 points all indicating a significant rise, that seems pretty trustworthy.  Does reducing `minRecency` to 15 minutes (or 20 minutes) result in too many false positives in some way?

Reply:
I think it's probably OK to set the `minRecency` to 15 minutes. Since the algorithm now looks for direct carb absorption, and the carb threshold is set higher than most noisy CGM jumps, I think this shouldn't result in too many false positives. I'll make the change and would lean towards getting some live testing before merging this to check my assumptions